Pods/Manifest.lock

```diff
@@ -24,7 +24,7 @@ DEPENDENCIES:
   - SwiftLint
 
 SPEC REPOS:
-  trunk:
+  https://cdn.cocoapods.org/:
```

Comment:
per tim's comment above -- same with this.. is this expected? i would have probably reverted this change.

Reply:
me too, but why did my cocoapods make that change? If that's not supposed to happen then I'm very confused 😕

Reply:
https://github.com/CocoaPods/CocoaPods/issues/9190